### DISCUSSION TITLE
How to set min-width in text preview dialog?

### DISCUSSION THREAD

**Discussion by efkz** [xdan/jodit#1124] [Q&A] [answered]
Now, it seems min-width is hardcoded with 1024px which leads to horizontal scroll

**Comment by xdan** [comment, marked as answer]
That width comes from CSS, so you can override it. The preview opens in a Jodit dialog, so target its panel in your stylesheet — something like `.jodit-dialog__panel { min-width: 0; width: 90vw; max-width: 900px; }` (bump specificity if your reset doesn't win). That drops the forced 1024px and the horizontal scroll.
